src/Pages/LabelPage/labelPage.css

```diff
@@ -0,0 +1,28 @@
+.label-main {
+  margin-top: 6rem;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+}
+
+.btn-label {
+  width: 10rem;
+  padding: 0.7rem 1rem;
+  border-radius: 3rem;
+  border: none;
+  cursor: pointer;
+  transition: all 0.5s ease;
+}
+
+.btn-label:hover {
+  background-color: var(--primary-color);
+  color: var(--white);
+}
+
+.selected {
+  background-color: var(--primary-color);
+  color: var(--white);
+}
```

Comment:
You can merge these two `.selected` and `.btn-label:hover`. Also if you are using this more often in your code, then give it a generic name and use it wherever you need it.